sdk/dotnet/Pulumi.Kubernetes.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageProjectUrl>https://pulumi.com</PackageProjectUrl>
     <RepositoryUrl>https://github.com/pulumi/pulumi-kubernetes</RepositoryUrl>
     <PackageIcon>logo.png</PackageIcon>
-    <Version>4.14.0-alpha.1718665781</Version>
+    <Version>4.0.0-alpha.0+dev</Version>
```

Comment:
Should I do something in CI here? Will this be done by some release process?

Reply:
We can ignore this change for now - but we should fix it in the future to prevent it from always changing. The final release version will be embedded in the release process, so it's irrelevant here.